charts/tfy-notebook-controller/templates/deployment.yaml

```diff
@@ -103,4 +103,8 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      affinity:
+        {{- with .Values.affinity }}
```

Comment:
Lets move this line above `affinity` like the previous key? This would prevent incorrect yaml in case of null affinity being supplied

Reply:
By default affinity is supplied `{}`.